align/main.py

```diff
@@ -287,7 +320,7 @@ def schematic2layout(netlist_dir, pdk_dir, netlist_file=None, subckt=None, worki
     if '1_topology' in steps_to_run:
         netlist = extract_netlist_files(netlist_dir,netlist_file)
         if subckt is None:
-            subckt = netlist.stem
+            subckt = netlist.stem.upper()
```

Comment:
Why do we need to do this? (And everywhere else.) If we have to do it, we should do it in one place.

Reply:
The new netlist parser from schema is case insensitive (similar to spice syntax assumption) and the idea is to run the whole flow in caps instead of worrying about case.